src/jdk.compiler/share/classes/com/sun/tools/javac/parser/DocCommentParser.java

```diff
@@ -287,6 +287,7 @@ protected DCTree blockTag() {
     }
 
     protected void inlineTag(ListBuffer<DCTree> list) {
+        assert buf[bp] == '{' : buf[bp];
```

Comment:
not sure that this is what you want to do. We usually use com.sun.tools.javac.util.Assert for this as they can't be skip if the code is not executed with assertions off. Same for similar changes in this patch

Reply:
I'm was hoping to use the `assert` facility for the purposes it was designed for. That particular occurrence of `assert` as well as others in this pull request are about asserting that **our code** is being correct rather than input being correct. The invariants those assertions establish MUST hold regardless of the input.

Reply:
right in that case you should certainly use com.sun.tools.javac.util.Assert instead

Reply:
I won't use the `com.sun.tools.javac.util.Assert` facility as it has a number of disadvantages over `assert` in that particular case. However, I will remove `assert` statements and update the pull request.

Will that do?